app/controllers/concerns/course/users_controller_management_concern.rb

```diff
@@ -86,7 +86,7 @@ def update_request_origin
 
   # Selects an appropriate redirect path depending on the user being deleted.
   def delete_redirect_path
-    if @course_user.staff?
+    if @course_user.teaching_staff?
```

Comment:
Should we keep this as `staff?`?

Since an observer is considered a staff member, if an observer is deleted, I think it should redirect to the staff listing.

Reply:
Good catch - thought this `@course_user` meant something else >_<